Tools-Override/publishtest.targets

```diff
@@ -9,7 +9,6 @@
 
   <PropertyGroup>
     <SerializeProjects Condition="'$(TestWithLocalLibraries)'=='true'">true</SerializeProjects>
-    <TargetOS Condition="'$(OSGroup)'!='AnyOS'">$(OSGroup)</TargetOS>
```

Comment:
@mellinoe we are going to likely need some more work around TargetOS to get this correctly setup for test builds on the build machines and test runs on a helix machine, if the build machine TargetOS doesn't match the machines we plan to run on.